apropos.html

```diff
@@ -0,0 +1,143 @@
+                    <p class="margin-propos bold">Antoine Bergeron<br>
+                        Québec, QC<br>
+                        antoine.bergeron1@hotmail.com<br>
+                        418-271-9618<br></p>
+                </div>
+            </div>
+
+            <h2 class="fw-bolder padding-h2">Informations personelles</h2>
+            <div class="bar-noir"></div>
+            <ul>
+                <li class="padding-projets bold">Diplôme le plus haut que vous ayez obtenu: DEP</li>
+                <li class="padding-propos bold">Fort intérêt pour les jeux vidéos - Monter des ordinateurs - Sports</li>
+                <li class="padding-projets-last bold">Étudiant au collège François-Xavier Garneau en développement web</li>
+
+            </ul>
+            <h2 class="fw-bolder padding-h2">Expériences professionnelles</h2>
+            <div class="bar-noir"></div>
+            <h4 class="fw-bolder padding-h2">Sous-chef au Sésame Lebourneuf de mai 2021 à actuellement</h4>
+            <ul>
+                <li>- Assurer le bon fonctionnement du service</li>
+                <li>- Donner des tâches</li>
+                <li>- Gérer les commandes</li>
+                <li>- Faire les horaires</li>
+            </ul>
+            <h4 class="fw-bolder padding-h2">Chef de cuisine à l'école mont saint-sacrement de août 2018 à mai 2021</h4>
+            <ul>
+                <li>- Monter le présentoire à dessert</li>
+                <li>- Préparer des souper table d'hôtes</li>
+                <li>- Gérer les commandes</li>
+            </ul>
+            <h4 class="fw-bolder padding-h2">Sous-chef chez Victor de janvier 2016 à juillet 2018</h4>
+            <ul>
+                <li>- Ouverture du restaurant</li>
+                <li>- Donner des tâches</li>
+                <li>- Gérer le fonctionnement du service</li>
+                <li>- Préparation des aliments</li>
+            </ul>
+
+            <h2 class="fw-bolder padding-h2">Formation</h2>
+            <div class="bar-noir"></div>
+            <ul>
+                <li class="padding-propos-formation bold">DEP en Cuisine </li>
+                <li class="padding-propos bold">Fierbourg - Ville de Québec, QC</li>
+                <li class="padding-projets-last bold">août 2018 - actuellement</li>
+            </ul>
+            <h2 class="fw-bolder padding-h2">Compétences</h2>
+            <div class="bar-noir"></div>
+            <ul>
+                <li class="padding-propos-formation">- Résolution de problèmes</li>
+                <li class="padding-propos">- Communication</li>
+                <li class="padding-propos">- Persévérence</li>
+                <li class="padding-projets-last">- Débrouillard</li>
+            </ul>
+
+            <div class="bar-noir"><a href="#"><img class="logo-arrow" src="/assets/arrow.png" alt="flèche vers le haut"></a>
+            </div>
+
+
+            <footer>
+
+
+                <p class="text-align-right">Haut de<br>page</p>
```

Comment:
Your footer have a lot of issues:

![image](https://user-images.githubusercontent.com/6362750/235274863-bb1b7d4a-f0aa-47d4-8c76-63367cb2594e.png)